trigger-pipeline/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.7-stretch
```

Comment:
should we use a newer distribution as base image

Reply:
Updated